man.rst

```diff
@@ -65,14 +65,16 @@ EXAMPLES
 
     __ungleich_nginx_app_proxy --config jetty --ssl --dh 2048
 
-    # Select certificate below /etc/ssl/certs/${ssl_name}.crt
-    # Select key below /etc/ssl/private/${ssl_name}.key
-    __ungleich_nginx_app_proxy --config jetty --dh 2048 \
-    --ssl --ssl-name test.example.org
-
+    # Select a certificate on your machine
     __ungleich_nginx_app_proxy --config rails --dh 2048 \
```

Comment:
how to solve?
a) write it into the man
b) check something like: 
`if [ ! -f "$__object/parameter/ssl" ] || [ -f "$__object/parameter/dh" ]; then
    echo "Missing param"
fi
`
I agree that is this nessecary for ssl. But maybe you have still a DH, so you dont need this.